pages/_app.js

```diff
@@ -1,6 +1,7 @@
-import "@brainhubeu/react-carousel/lib/style.css";
 import "react-toastify/dist/ReactToastify.css";
 import "public/scss/main.scss";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
```

Comment:
```suggestion
import "slick-carousel/slick/slick.css";
import "slick-carousel/slick/slick-theme.css";
import "public/scss/main.scss";
```

alangkah bijaknya jika css yang buat slick-nya ditaro sebelum yang main.scss